File: Basics_with_JavaScript/56 Conditional Statements/More Exersises/01. Pipes In Pool.js
function vladi(input){

    let poolCapacity = Number(input[0])
    let tube1Capacity = Number(input[1])
    let tube2Capacity = Number(input[2])
    let hours = Number(input[3])

    currentVolume = tube1Capacity * hours + tube2Capacity * hours
    percentage = (100 * currentVolume / poolCapacity).toFixed(2)
    percentageTube1 = (100 * tube1Capacity * hours / currentVolume).toFixed(2)
    percentageTube2 = 100 - percentageTube1
    difference = Math.abs(currentVolume - poolCapacity)

    if (currentVolume <= poolCapacity) {
        console.log(`The pool is ${percentage}% full. Pipe 1: ${percentageTube1}%. Pipe 2: ${percentageTube2}%.`);
    } else {
        console.log(`For ${hours} hours the pool overflows with ${difference} liters.`);
    }

    // console.log(percentageTube1);
    // console.log(percentageTube2);
    // console.log(percentage);
    
    };
   
    vladi([1000, 100, 120, 3]);
    vladi([100, 100, 100, 2.5]);

File: Basics_with_Python/PythonBasicCourse/2_Conditional_Statements/6_Conditional_Statements_Excercises___Exercise/more_exercises/01._Pipes_In_Pool.py
import math
volume = int(input())
p1 = int(input())
p2 = int(input())
time = float(input())

p1 = p1 * time
p2 = p2 * time
tot = p1 + p2
totPercent = 0

if volume >= tot:
    totPercent = (100 * tot / volume)
    p11 = p1 * 100 / tot
    p22 = p2 * 100 / tot

    print(f"The pool is {totPercent:.2f}% full. Pipe 1: {p11:.2f}%. Pipe 2: {p22:.2f}%.")
else:
    totPercent = (tot - volume)
    print(f"For {time} hours the pool overflows with {totPercent:.2f} liters.")
